core/src/main/java/nl/nn/adapterframework/filesystem/FileSystemActor.java

```diff
@@ -579,53 +581,58 @@ protected void addActions(List<FileSystemAction> specificActions) {
 		actions.addAll(specificActions);
 	}
 
-	@IbisDoc({"1", "If parameter ["+PARAMETER_ACTION+"] is set, then the attribute action value will be overridden with the value of the parameter.", "" })
+	@IbisDoc({"If parameter ["+PARAMETER_ACTION+"] is set, then the attribute action value will be overridden with the value of the parameter.", "" })
 	public void setAction(FileSystemAction action) {
 		this.action = action;
 	}
 
-	@IbisDoc({"2", "Folder that is scanned for files when action="+ACTION_LIST+". When not set, the root is scanned", ""})
+	@IbisDoc({"Folder that is scanned for files when action="+ACTION_LIST+". When not set, the root is scanned", ""})
 	public void setInputFolder(String inputFolder) {
 		this.inputFolder = inputFolder;
 	}
 
-	@IbisDoc({"3", "when set to <code>true</code>, the folder to move or copy to is created if it does not exist", "false"})
+	@IbisDoc({"If set <code>true</code>, the folder to move or copy to is created if it does not exist", "false"})
 	public void setCreateFolder(boolean createFolder) {
 		this.createFolder = createFolder;
 	}
 
-	@IbisDoc({"4", "when set to <code>true</code>, for actions "+ACTION_MOVE+", "+ACTION_COPY+" or "+ACTION_RENAME+", the destination file is overwritten if it already exists", "false"})
+	@IbisDoc({"If set <code>true</code>, for actions "+ACTION_MOVE+", "+ACTION_COPY+" or "+ACTION_RENAME+", the destination file is overwritten if it already exists", "false"})
 	public void setOverwrite(boolean overwrite) {
 		this.overwrite = overwrite;
 	}
 
-	@IbisDoc({"5", "filename to operate on. When not set, the parameter "+PARAMETER_FILENAME+" is used. When that is not set either, the input is used", ""})
+	@IbisDoc({"If set <code>false</code>, for actions "+ACTION_MOVE+" or "+ACTION_COPY+", the destination file is not guaranteed to be returned, allowing to move and copy files for destinations where the result file is not known", "true"})
+	public void setDestinationFileMustBeReturned(boolean destinationFileMustBeReturned) {
+		this.destinationFileMustBeReturned = destinationFileMustBeReturned;
+	}
```

Comment:
Dit is meer voor java ontwikkelaars toch?

Reply:
Nee, dit is voor Frank developers, die met een ExchangeFileSystemPipe een file willen moven naar een andere mailbox, en dan geen exceptie willen krijgen. Die kunnen hem dan op false zetten.

Reply:
Ik weet niet of het handig is hier een attribuut voor te maken, ik zou verwachten dat deze automatisch gezet wordt, en anders altijd aan staan.

Reply:
Het staat inderdaad altijd aan, maar je kan het juist uit zetten, als je weet dat je een exhange bericht naar een andere mailbox gaat moven, en dan weet dat je geen resultaat terug krijgt, maar dat niet erg vindt